### ISSUE TITLE
VERSION is still at 1.0.1

### ISSUE THREAD

**Issue by JacobAMason** [jenkins-x/jx#1207]
Building jx from the latest source always yields a binary which believes it is version 1.0.1 because the VERSION file is behind. Is there a good reason for this?

**Comment by i0n** [comment]
Hello!

I believe this is because the VERSION currently gets injected into the binary as part of the build process, so any self compiled binary will always use 1.0.1.

Maybe there's a better way of doing this, any ideas?

**Comment by JacobAMason** [comment]
That (version string getting injected) was my assumption, seeing the Jenkins bot issuing releases. I hope the release process is powered by Jenkins-x, because that'd be cool.

I'm assuming there is a one-liner that the bot is executing to build this that includes the newest version. My recommendation is to disclose a similar `make` command in the build-from-source instructions that injects a version number (`export VERSION=` or whatnot), and then remove the VERSION file--I'm guessing it's completely ignored by CD, is never incremented by anyone making PRs, and is not going to be noticed by most people building from source until they run `jx version` and get an auto update warning.

As an aside, auto update is neat, but might make this hard to be used with a package manager. The reason I stumbled onto this was because I wasted to install jx through the aur and noticed it didn't exist so I've been trying to set up the package at https://aur.archlinux.org/packages/jx/ I'm not entirely sure how to resolve the two different update strategies and am open to suggestions and assistance on that.
